src/browser/Sync.js

```diff
@@ -0,0 +1,20 @@
+function notSupported() {
+    console.log('ContentSync is not supported on browser platform');
+}
+
+var ContentSync = function() {};
+ContentSync.prototype.on = function() {};
+ContentSync.prototype.emit = function() {};
+ContentSync.prototype.cancel = function() {};
```

Comment:
Yes. Because those are all called from the `.sync` method, if they also included the `notSupported()` function the error message gets logged twice for each call